chminfo_service/LICENSE

```diff
@@ -0,0 +1,23 @@
+The MIT License (MIT)
+
+Copyright (c) 2015, The MITRE Corporation. All rights reserved.
```

Comment:
I wrote this code. Do you have any recommendations on license agreements? If I select a slightly different license does that cause any issues for you; should I just use MIT?